src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Content from "./components/Content";
+import Sider from "./components/Sider";
+import Header from "./components/Header";
+import { ContentWrapper } from "./app.style";
+
+function App() {
+  return (
+    <div>
+      <Header />
+      <ContentWrapper>
+        <Sider />
+        <Content />
```

Comment:
I would prefer to keep all the routes in some root component like `App` or `Router`, where everybody can immediately see the app structure. `Content` in components looks more like some reusable wrapper for the content.

Reply:
DONE